src/System.Buffers.Primitives/System.Buffers.Primitives.csproj

```diff
@@ -8,10 +8,8 @@
     <PackageIconUrl>http://go.microsoft.com/fwlink/?linkid=833199</PackageIconUrl>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="System.Buffers" Version="$(CoreFxVersion)" />
+    <PackageReference Include="System.Buffers" Version="$(CoreFxStableVersion4_4)" />
```

Comment:
Question: For a netstandard library, like System.Buffers.Primitives for example, is it better to reference a lower version of a package or higher (4.4.0 vs 4.3.0)? This change results in us referencing the higher version.

cc @ericstj

Reply:
Typical best practice is to reference the lowest version that satisfies your dependency (both direct and indirect).  Be careful not to reference a version lower than any of your other dependencies or you will cause a downgrade.

There are sometimes requirements that folks have (limiting test matrix, or limiting number of packages downloaded) that will cause them to break from this best practice and reference higher versions then they actually need, so don't be surprised if you find that this advice is not followed consistently.